### PR TITLE
fix: [AXIMST-488-489] Textbooks: file size and upload connection alert

### PR DESCRIPTION
**Pull request by vladislavkeblysh**
- Fixed upload file max size
- Fixed internet connection alert

[AXIMST-488](https://youtrack.raccoongang.com/issue/AXIMST-488/upload-The-error-message-doesnt-appear-when-the-Internet-connection-is-lost)
[AXIMST-489](https://youtrack.raccoongang.com/issue/AXIMST-489/uploadlimit-there-is-no-message-if-a-user-upload-a-large-file)

### CODE REVIEW

**Review by khudym — APPROVED** [review]

File: src/textbooks/Textbooks.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
+@import "./scss/variables";
 @import "./empty-placeholder/EmptyPlaceholder";
 @import "./textbook-card/TextbookCard";
 @import "./textbook-form/TextbookForm";
+
+.alert-toast {
+  z-index: $popover !important;
```
Comment (PKulkoRaccoonGang) [inline src/textbooks/Textbooks.scss:7]: [important]: Try to use Paragon SCSS variable

```suggestion
  z-index: $zindex-tooltip !important;
```

File: src/textbooks/textbook-form/TextbookForm.jsx [hunk 2]
```diff
@@ -23,6 +23,7 @@ import ModalDropzone from '../../generic/modal-dropzone/ModalDropzone';
 import { useModel } from '../../generic/model-store';
 import textbookFormValidationSchema from './validations';
 import messages from './messages';
+import { UPLOAD_FILE_MAX_SIZE } from '../../constants';
```
Comment (PKulkoRaccoonGang) [inline src/textbooks/textbook-form/TextbookForm.jsx:26]:
```suggestion
import { useModel } from '../../generic/model-store';
import { UPLOAD_FILE_MAX_SIZE } from '../../constants';
import textbookFormValidationSchema from './validations';
import messages from './messages';
```

File: src/textbooks/Textbooks.scss (hunk 1 above)
Comment (khudym) [reply src/textbooks/Textbooks.scss:7]: @PKulkoRaccoonGang We have the same variable for certificates page, should we change there also?

Comment (khudym) [reply src/textbooks/Textbooks.scss:7]: i remember that you suggested exactly this naming so im curious

**Review by monteri — APPROVED** [review]
LGTM, resolve comments and ready to merge

File: src/textbooks/Textbooks.scss (hunk 1 above)
Comment (vladislavkeblysh) [reply src/textbooks/Textbooks.scss]: Fixed

File: src/textbooks/textbook-form/TextbookForm.jsx (hunk 2 above)
Comment (vladislavkeblysh) [reply src/textbooks/textbook-form/TextbookForm.jsx]: Fixed

**Review by monteri — APPROVED** [review]

**Pull request merged**